bin/prepare_project_edition.sh

```diff
@@ -76,6 +108,10 @@ docker-compose up -d
 echo '> Change ownership of files inside docker container'
 docker-compose exec app sh -c 'chown -R www-data:www-data /var/www'
 
+# Rebuild container
+docker-compose exec --user www-data app sh -c "rm -rf var/cache/*"
```

Comment:
The Symfony container was build using the `install_dependencies` (Docker) container, we need to rebuild it inside the `app` container which has all the variables defined.